src/pages/EnablePayments/ActivateStep.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable max-len */
+import React from 'react';
+import {View, Text} from 'react-native';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import styles from '../../styles/styles';
+import userWalletPropTypes from './userWalletPropTypes';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+    ...userWalletPropTypes,
+};
+
+const defaultProps = {
+    userWallet: {},
+};
+
+const ActivateStep = props => (
+    <ScreenWrapper>
+        <HeaderWithCloseButton
+            title={props.translate('activateStep.headerTitle')}
+            onCloseButtonPress={() => Navigation.dismissModal()}
```

Comment:
NAB
```suggestion
            onCloseButtonPress={Navigation.dismissModal}
```

Reply:
I had thought about this too... however, there is an event that is passed as the first param to the `onCloseButtonPress` callback, and I don't think we want that being passed to `dismissModal()`